extras/create_rcsinfo.sh

```diff
@@ -6,7 +6,7 @@
 
 # default value, if git is not installed or if the source directory is
 # not under version control
-version="v2.10.0"
+version="v3.1.0-SNAPSHOT"
```

Comment:
Shouldn't this be `v3.0.0-SNAPSHOT`? And wouldn't it be better to build this string automatically from `NEST_VERSION_VERSION` defined in `CMakeLists.txt`?